conf/messages.en

```diff
@@ -951,6 +951,12 @@ confirmation.validationError.table.caption = Errors
 confirmation.validationError.table.head1 = Section
 confirmation.validationError.table.head2 = Error
 
+replacementDisclosureConfirmation.title = Replacement received
+replacementDisclosureConfirmation.p1 = We have sent a confirmation email to example@example.com.
```

Comment:
Should the email be shown in the confirmation?

Reply:
Email sentence has been removed temporarily